templates/suspenders_layout.html.erb.erb

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8" />
   <meta name="ROBOTS" content="NOODP" />
   <meta name="viewport" content="initial-scale=1" />
+  <!-- change the title in config/locales/en.yml in the en.titles key -->
```

Comment:
:-1: I like this gem. It works great for setting custom titles depending on your current page.

I don't think that it is fair to say it is confusing, the usage is just a bit hidden and pointing it out would be a sufficient way to fix the problem.